src/unix/pipe.c

```diff
@@ -383,6 +383,9 @@ static int uv__pipe_getsockpeername(const uv_pipe_t* handle,
     addrlen = p - sa.sun_path;
   }
 
+  if (buffer == NULL || size == NULL)
+    return UV_EINVAL;
```

Comment:
Maybe do this first thing first, before making syscalls, etc.?

Reply:
Done!